main.ts

```diff
@@ -87,8 +87,16 @@ export default class ReadwiseMirror extends Plugin {
   headerTemplate: Template;
   highlightTemplate: Template;
 
-  private formatTags(tags: Tag[]) {
-    return tags.map((tag) => `#${tag.name}`).join(', ');
+  private formatTags(tags: Tag[], nohash: boolean = false) {
+    // use unique list of tags
+    const uniqueTags = [...new Set(tags.map((tag) => tag.name.replace(/\s/, '-')))];

```

Comment:
Readwise might return tags with whitespace. These are incompatible with Markdown and Obsidian and therefore need to be sanitized. Replacing whitespace with a dash (-) seems a sensible choice that should not break frontmatter.